flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -1879,4 +1881,17 @@ private Stream<TestSetSpec> splitTestCases() {
                                 "SPLIT(f1, '1', '2')",
                                 "No match found for function signature SPLIT(<CHARACTER>, <CHARACTER>, <CHARACTER>)"));
     }
+
+    private Stream<TestSetSpec> arrayElementTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.ARRAY_ELEMENT)
+                        .onFieldsWithData(new Integer[] {1}, new Integer[] {1, 2})
+                        .andDataTypes(
+                                DataTypes.ARRAY(DataTypes.INT()), DataTypes.ARRAY(DataTypes.INT()))
+                        .testResult($("f0").element(), "ELEMENT(f0)", 1, DataTypes.INT())
+                        .testSqlRuntimeError(
+                                "ELEMENT(f1)",
+                                TableRuntimeException.class,
+                                "Array has more than one element."));
```

Comment:
great, 
one last comment: can we move all the test for `element` function from `ArrayTypeTest#testArrayOperations` here?

It would simplify future code navigation

Reply:
done!